multiselectfield/db/fields.py

```diff
@@ -54,14 +55,17 @@ def __init__(self, *args, **kwargs):
         self.max_choices = kwargs.pop('max_choices', None)
         super(MultiSelectField, self).__init__(*args, **kwargs)
         self.max_length = get_max_length(self.choices, self.max_length)
-        self.validators[0] = MaxValueMultiFieldValidator(self.max_length)
+        self.validators.append(MaxValueMultiFieldValidator(self.max_length))
```

Comment:
Not sure the best place to mention this, but I was working on this last year, and this changes how the validators work.

The original removes/replaces the existing CharField MaxLengthValidator. 

Appending to the validators means that both the CharField MaxLengthValidator and MaxValueMultiFieldValidator are present.    Previously if you specified your model without a max_length the max length would be set to the default 200 in get_max_legnth.  This causes get a validation error when django.core.validators MaxLengthValidator.compare tries to compare against None.

I changed my fork to use:
```
self.validators = [v for v in self.validators if not isinstance(v, validators.MaxLengthValidator)]
self.max_length = get_max_length(self.choices, self.max_length)
self.validators.append(MaxValueMultiFieldValidator(self.max_length))
```

Reply:
Can you submit a PR?

Reply:
Sure thing.